api_service.py

```
@@ -1542,14 +1545,112 @@ async def trigger_monthly_summary(x_api_key: Optional[str] = Header(None)):
+
+        return OAuthStateInitResponse(
+            success=True,
+            state_token=request_body.state_token,
+            expires_at=expires_at_iso
+        )
+
+    except Exception as e:
+        logger.exception("Failed to store OAuth state token")
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail=f"Failed to register state token: {str(e)}"
```

Ruff (0.14.10)

Use explicit conversion flag

Replace with conversion flag

(RUF010)